### PR TITLE
ENH try osx

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@chrisburr The osx build of the python extension failed with a symbol not found (`_lua_strlen`). Do you know how to fix this?

I commented out the python extension build on osx for now.

**Comment by chrisburr** [comment]
No idea I was hoping you might 😄

Regardless I think it's fine to merge with them disabled.

**Comment by beckermr** [comment]
I assume this has to do with getting liblua.a linked into the python extension when it is built. We could try a patch at another time.

**Comment by beckermr** [comment]
ok @chrisburr this one is ready for review!

File: recipe/build.sh [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
 
 declare -a CONFIGURE_ARGS
-if "${PYTHON}" --version | grep -i pypy; then
-    CONFIGURE_ARGS+=("PYTHON_CFLAGS=-I$CONDA_PREFIX/include")
-    CONFIGURE_ARGS+=("PYTHON_LIBS=-L$PREFIX/lib -lpypy3-c")
+if [[ ${target_platform} == "osx-64" ]]; then
+  # CONFIGURE_ARGS+=("--disable-dependency-tracking")
+  # CONFIGURE_ARGS+=("--disable-silent-rules")
+  # CONFIGURE_ARGS+=("--enable-nls")
+  # CONFIGURE_ARGS+=("--disable-plugins")
```
Comment (beckermr) [inline recipe/build.sh:8]: These extra flags are in the homebrew recipe for osx. IDK if we want them here.

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
Thanks @beckermr!

**Pull request merged**